components/Tabs/Tabs.js

```diff
@@ -1,24 +1,27 @@
-
 class TabItem {
   constructor(element) {
     // attach dom element to object. Example in Tabs class
+    this.element = element;
   }
 
   select() {
     // should use classList
+    const item = document.querySelector('.List .List__item'); // `.querySelector` selects the class `.List` wich holds another class, `.List__item` which is the class attached to the 3 buttons (tabs) that we want. We are attaching this result to the `const item` so we can easily use it in the line below.
```

Comment:
element.classList is a read-only property that contains the list of class attributes that an element, or node, contains.
You used classList.remove() below. The same thing can be done here.  
```
const item = this.element.classList.add('[insert name of class to add here]')
```